### DISCUSSION TITLE
Creating Plays - Using Filter Option

### DISCUSSION THREAD

**Discussion by tempest3991** [Security-Onion-Solutions/securityonion#7152] [Unsupported Versions]
Hello, 

We have a specific server that executes a whoami command as part of its design, so it is getting caught up in our alerts. I'd like to edit the playbook rule to filter out this server if possible, but I can't find any documentation on how to filter out rules. I've found snippets that tell me to "just use the filter command" but without any documentation or examples. 

I've copied the play, but don't know how to add the filter: context with the IP address, I want to filter out a specific IP address from alerts if this is possible. 

Thanks for any help!

Here is the Play below:

title: Whoami Execution
id: e28a5a99-da44-436d-b7a0-2afc20a5f413
status: experimental
description: Detects the execution of whoami, which is often used by attackers after
  exloitation / privilege escalation but rarely used by administrators
references:
- https://brica.de/alerts/alert/public/1247926/agent-tesla-keylogger-delivered-inside-a-power-iso-daa-archive/
- https://app.any.run/tasks/7eaba74e-c1ea-400f-9c17-5e30eee89906/
author: Florian Roth
date: 2018/08/13
tags:
- attack.discovery
- attack.t1033
- car.2016-03-001
logsource:
  category: process_creation
  product: windows
detection:
  selection:
    Image: '*\whoami.exe'
  selection2:
    OriginalFileName: whoami.exe
  condition: selection or selection2
falsepositives:
- Admin activity
- Scripts and administrative tools used in the monitored environment
level: high

**Comment by defensivedepth** [comment]
So first off, we would need to filter it using a field that is existing in the source log - in this case, a process_creation log.

Would hostname work?
